src/utils/createMusicMetadata.ts

```diff
@@ -61,35 +63,33 @@ const music_metadata = {
 	},
 }
 
-const createMusicMetadata = data => {
-	music_metadata.name = data.name
-	music_metadata.title = data.name
-	music_metadata.attributes.project = data.name
-	music_metadata.tags = [data.name, data.address]
-	music_metadata.artist = data.address
-	music_metadata.credits = [{ name: data.address, collaboratorType: 'creator' }]
-	music_metadata.image = data.image
-	music_metadata.artwork.uri = data.image
-	music_metadata.animation_url = data.song
-	music_metadata.losslessAudio = data.song
-	music_metadata.description = data.description
-	const project = {
-		title: data.name,
+const createMusicMetadata = ({artist, name, description, song, image}: FormData) => {
+	const metadata = {
```

Comment:
no, this totally breaks the music metadata standard...

if you merge this I will fork the codebase this is not a valid music nft

Reply:
you realize this is a totally different metadata than what we're using now correct?

Reply:
yeah, your right. I'll put the empty fields back in